modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -156,84 +177,145 @@ class NDB_Form_Imaging_Browser extends NDB_Form
-                'QCStatus' => $FileObj->getParameter('QCStatus'),
-                'QCDate' => $FileObj->getParameter('QCLastChangeTime'),
-                'Caveat' => $FileObj->getParameter('Caveat'),
-                'SNR' => $FileObj->getParameter('SNR'),
-                'SeriesUID' => $FileObj->getParameter('SeriesUID'),
-                'XMLreport' => $FileObj->getParameter('processing:DTIPrepXmlReport'),
-                'XMLprotocol' => $FileObj->getParameter('ProtocolFile'),
-                'NrrdFile' => $FileObj->getParameter('processing:nrrd_file'),
-                'OtherTimepoints' => $this->_getFilesAcrossTimepoints((int)$FileObj->getParameter('SessionID'),
-                    (int)$FileObj->getParameter('AcquisitionProtocolID'))
+
+            $FileID       = $fileRow['FileID'];
+            $Filename     = basename($FileObj->getParameter('File'));
+            $CheckPic     = "/mri/jiv/get_file.php?file=pic/".
+                                         $FileObj->getParameter(
+                                             'check_pic_filename'
+                                         );
+            $FullFilename = $FileObj->getParameter('File');
+            $JivFilename  = basename($FileObj->getParameter('File'));
+            $JivAddress   = str_replace(
+                '_check.jpg',
+                '',
+                $FileObj->getParameter('check_pic_filename')
+            );
+            $New          = ($FileObj->getParameter('QCFirstChangeTime') == '') ?
+                                         1 : 0;
+            $Pipeline     = $FileObj->getParameter('Pipeline');
+            $OutputType   = $FileObj->getParameter('OutputType');
+            $AcquisitionProtocol = $FileObj->getAcquisitionProtocol();
+            $CoordinateSpace     = $FileObj->getParameter('CoordinateSpace');
+            $Algorithm           = $FileObj->getParameter('Algorithm');
+            $AcquisitionDate     = $this->_getDate(
+                $FileObj,
+                'acquisition_date',
+                $acqDate
+            );
+            $ProcDate            = $this->_getDate(
+                $FileObj,
+                'processing:processing_date',
+                $procDate
+            );
+            $FileInsertDate      = $FileObj->getParameter('InsertTime');
+            $SeriesDescription   = $FileObj->getParameter('series_description');
+            $SeriesNumber        = $FileObj->getParameter('series_number');
+            $EchoTime            = number_format(
+                $FileObj->getParameter('echo_time')*1000,
+                2
+            );
+            $RepetitionTime      = number_format(
+                $FileObj->getParameter('repetition_time')*1000,
+                2
+            );
+            $SliceThickness      = number_format(
+                $FileObj->getParameter('slice_thickness'),
+                2
+            );
+            $Time    = number_format($FileObj->getParameter('time'), 2);
```

Comment:
@kongtiaowang @driusan 

This pull requests seems to have made the entries: EchoTime, RepetitionTime, SliceThickness, and Time fields empty in the Header of the image in Imaging Browser.